beacon/src/avahi/resolving.rs

```diff
@@ -15,40 +12,67 @@ pub struct Resolved {
     pub txt: HashMap<String, String>,
 }
 
-impl TryFrom<FoundArgs<'_>> for Resolved {
+impl
+    TryFrom<(
+        i32,          //interface 0
+        i32,          //protocol 1
+        String,       //name 2
+        String,       //service_type 3
+        String,       //domain 4
+        String,       //host 5
+        i32,          //approtocol 6
+        String,       //address 7
+        u16,          //port 8
+        Vec<Vec<u8>>, //txt 9
+        u32,          //flags 10
+    )> for Resolved
```

Comment:
This is a long, complex type. A type alias might be better than bundling all these values in the impl. I wish this was a struct. But I can't blame zbus for that.